my-app/src/MenuItems.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import CreateCategory from './CreateCategory.js';
+
+export default function MenuItems(props) {
+    const categories = props.categories;
+    return (
+    <div className='menu-items col-4'>
+        {categories.map(item => {
+            return <CreateCategory categoryName = {item.displayName}/>
```

Comment:
Whenever rendering an array of sibling elements, you need to give a unique `key` to each element.

Read about this concept here: https://reactjs.org/docs/lists-and-keys.html